### ISSUE TITLE
After upgrading from 2.19.0 to 2.21.0, pulumi preview(or up) throws error

### ISSUE THREAD

**Issue by johnwook** [dirien/pulumi-vultr#355]
```
Diagnostics:
  pulumi:providers:vultr (default_2_21_0_github_/api.github.com/dirien/pulumi-vultr):
    error: could not read plugin [$HOME/.pulumi/plugins/resource-vultr-v2.21.0/pulumi-resource-vultr] stdout: EOF

  pulumi:pulumi:Stack (theysends-infra-prod):
    error: fatal: failed to Init GRPC to register RPC handlers: failed to create resource provider: info.P must be constructed with ShimProvider or ShimProviderWithContext: found non-conforming type *muxer.ProviderShim

```

Downgrading from 2.21.0 to 2.19.0 fixed the issue.

Is there anything that I could be a help to fix the issue?

**Comment by dirien** [comment]
Hey @johnwook,

thanks for letting me know. There must have been a little bug. I fixed it in version `v2.21.1`. Please upgrade and try again!

**Comment by johnwook** [comment]
Thanks for your quick response :) It's working with verison of `v2.21.1`
